csaf_2.1/prose/edit/src/distributing-04-transition-20-21.md

```diff
@@ -0,0 +1,74 @@
+## Transition between CSAF 2.0 and CSAF 2.1
+
+This subsection details the process that SHOULD be followed when transitioning the distribution of documents from CSAF 2.0 to CSAF 2.1.
+Different scenarios can be encountered:
+
+- Providers will continue to use CSAF 2.0.
+  This SHOULD be avoided, as the CSAF ecosystem greatly profits from adoption to the new standard version.
```

Comment:
Shouldn't that be "adoption of" instead of "adoption to"?
```suggestion
  This SHOULD be avoided, as the CSAF ecosystem greatly profits from adoption of the new standard version.